docs/integrations/sources/instagram.md

```diff
@@ -86,7 +86,8 @@ AirbyteRecords are required to conform to the [Airbyte type](https://docs.airbyt
 ## Changelog
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                   |
-| :------ | :--------- | :------------------------------------------------------- | :------------------------------------------------------------------------------------------------------------------------ |
+|:--------|:-----------|:---------------------------------------------------------|:--------------------------------------------------------------------------------------------------------------------------|
+| 1.0.12  | 2023-10-30 | [31947](https://github.com/airbytehq/airbyte/pull/31947) | Capture media_product_type column                                                                                         |
```

Comment:
Be more specific `Capture media_product_type column in media and stories stream`
Please bump the metadata and dockerfile version too.

Reply:
@marcosmarxm Made the requested changes!